source/html/renderer/src/card-elements.ts

```diff
@@ -1574,7 +1598,9 @@ class ActionCollection {
         this._expandedAction = null;
     }
 
-    private showActionCardPane(action: ShowCardAction) {
+    showActionCardPane(action: ShowCardAction | HttpAction) {
```

Comment:
This needs to remain private and it shouldn't accept an HttpAction. HttpActions can't be expanded. Card expansion and status reporting should be separate things.